portfolio/src/main/webapp/script.js

```diff
@@ -12,35 +12,80 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+const myName = 'Evan Saracay';
+
+class Typer {
+  constructor() {
+    this.txt = 'Evan Saracay';
+    this.deleting = true;
+    this.speed = 300;
+  }
+  backSpace() {
+    this.txt = myName.substring(0, this.txt.length - 1);
+    this.speed = 100;
+  }
+  type() {
+    this.txt = myName.substring(0, this.txt.length + 1);
+    this.speed = Math.random() * 100 / 2 + 200;
+  }
+}
+
+function txtRotate(text) {
+  if (text.deleting) {
+    text.backSpace();
+  } else {
+    text.type();
+  }
+
+  if (!text.deleting && text.txt === myName) {
+    text.deleting = true;
+  } else if (text.deleting && text.txt === 'Evan') {
+    text.deleting = false;
+  }
+  console.log(text.txt);
+  document.getElementById('intro_content').innerText = text.txt;
+}
+
+
+window.onload =
+    function() {
+  // Function to run on loading which is typing
+  mytyper = new Typer();
+  setInterval(txtRotate, mytyper.speed, mytyper);
 }
 
-function randomQuote() {
-  const quotes =
+/*This function picks a random movie from our list and displays
```

Comment:
Omit "this function". Also, `/*` should be `/**` and have a newline after it. Needs a space before `*/`. Docstrings (a comment describing what a function does) also should end with a period. See https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports for more details on how to write docstring.